kiss

```diff
@@ -619,7 +618,7 @@ pkg_build() {
     # If an explicit package is a dependency of another explicit package,
     # remove it from the explicit list as it needs to be installed as a
     # dependency.
-    for pkg do
+    for pkg in $explicit; do
         contains "$deps" "$pkg" || explicit2=" $explicit2 $pkg "
     done
     explicit=$explicit2
```

Comment:
There's still something funky happening here. ie, the total package count can mismatch if `contains` returns `true`. I need to look into it further as initial attempts to solve it failed.

Reply:
See amended commit message.